demo/src/commonTest/kotlin/arithmetic/ArithmeticTest.kt

```diff
@@ -1,13 +1,13 @@
-package me.alllex.parsus.demo
+package me.alllex.parsus.demo.arithmetic
 
-import me.alllex.parsus.demo.Expr.*
+import me.alllex.parsus.demo.arithmetic.Expr.*
 import kotlin.test.Test
 import kotlin.test.assertEquals
 
 class ArithmeticTest {
 
     @Test
-    fun `Expr parsing`() {
+    fun exprParsing() {
```

Comment:
Is this a KMP limitation that test (function?) names with spaces in them are not supported?

Reply:
Yeah, Kotlin/JS doesn't support spaces. One option is to use the annotation `@JsName("exprParsing")` (which I did do in one of the tests), but it ends up being a bit repetitive.